projects/vdk-plugins/vdk-meta-jobs/README.md

```diff
@@ -114,6 +116,33 @@ def run(job_input: IJobInput) - > None:
     MetaJobInput().run_meta_job(JOBS_RUN_ORDER)
 ```
 
+
+### Runtime sequencing
+
+The depends_on key stores the dependencies of each job - the jobs that have to finish before it starts.
+The DAG execution starts from the jobs with empty dependency lists - they start together in parallel.
+But what happens if they are too many? It could cause server overload. In order to avoid such unfortunate situations,
+a limit in the number of concurrent running jobs is set. This limit is
+a [configuration variable](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_configuration.py#L87)
+that you are able to set according to your needs. When the limit is exceeded, the execution of the rest of the jobs
+is not cancelled but delayed until a spot is freed by one of the running jobs. What's important here is that
+although there are delayed jobs due to the limitation, the overall sequence is not broken.
+
+
+### Data Job start comparison
+
+There are 3 types of jobs right now in terms of how are they started.
+
+* Started by Schedule
+   * When the time comes for a scheduled execution of a Data Job, if the one is currently running, it will be waited
+     to finish by retrying a few times. If it is still running then, this scheduled execution will be skipped.
```

Comment:
nit: may expect to have this already stated somewhere else (and linked then extended here), since fundamental about data jobs execution runtime

Reply:
I can't seem to find such bit in the current docs or in the code docs. Should I leave it this way?

Reply:
Leaving it this way. Might address this when we have the relevant docs somewhere else.